ansible/requirements.txt

```
@@ -1 +1 @@
-ansible==6.5.0
+ansible==7.0.0
```

Curieux, la dernière vraie version est 7.1.0

https://pypi.org/project/ansible/

```suggestion
ansible==7.1.0
```